app/vmui/packages/vmui/src/components/Main/Pagination/style.scss

```diff
@@ -0,0 +1,64 @@
+@use "../../../styles/variables" as *;
+
+.vm-pagination {
```

Comment:
Pagination is hardly visible in dark theme
<img width="401" alt="image" src="https://github.com/user-attachments/assets/8e05025b-b3d9-4a60-8bbf-396b2c838b4c">

Reply:
<img width="401" alt="image" src="https://github.com/user-attachments/assets/23b2479a-11d9-4cde-a7a4-26248aa1df56">